### PR TITLE
Replace Rimble-UI components with Material-UI

### PR DESCRIPTION
**Pull request by macfarla**

### CODE REVIEW

File: src/components/AccountTab/TableHeader.tsx [hunk 1]
```diff
@@ -11,16 +10,18 @@ type TableHeader = {
...
-      <Heading.h2 fontWeight="700">Accounts ({number})</Heading.h2>
-    </Box>
-    <Flex alignItems="center">
-      <Button icon="AddCircleOutline" onClick={() => openAddModal()} disabled={disabledAdd}>
-        Add Account
-      </Button>
-    </Flex>
-  </Flex>
+  <Grid container alignItems="center" justifyContent="space-between">
+    <Typography variant="h4">Accounts ({number})</Typography>
```
[the remark this answers is not in the document]
Comment (macfarla) [reply src/components/AccountTab/TableHeader.tsx]: fixed

File: src/components/Dashboard/TabSelector.tsx [hunk 2]
```diff
@@ -23,12 +22,10 @@ const TabSelector: React.FC<TabSelector> = ({ tab, setTab }) => (
         px={3}
         onClick={() => setTab(id)}
       >
-        <Text fontWeight="600" textAlign="center">
-          {text}
-        </Text>
+        <Typography variant="body1">{text}</Typography>
```
[the remark this answers is not in the document]
Comment (macfarla) [reply src/components/Dashboard/TabSelector.tsx]: changed to subtitle

File: src/components/EnodeTab/Row.tsx [hunk 3]
```diff
@@ -31,69 +30,43 @@ const EnodeRow: React.FC<EnodeRow> = ({
...
+      ) : status === PENDING_REMOVAL ? (
+        <Chip color="secondary" className={styles.pill} label="Pending Removal" />
+      ) : status === FAIL_ADDITION ? (
+        <Grid container>
+          <Chip color="secondary" className={styles.pill} label="Addition Failed" />
+          <Chip color="secondary" className={styles.pill} onClick={() => deleteTransaction(identifier)} label="Clear" />
+        </Grid>
+      ) : status === FAIL_REMOVAL ? (
+        <Grid container>
+          <Chip color="secondary" className={styles.pill} label="Removal Failed" />
```
[the remark this answers is not in the document]
Comment (macfarla) [reply src/components/EnodeTab/Row.tsx]: changed to default (light grey)

File: src/components/Toasts/Error.tsx [hunk 4]
```diff
@@ -13,15 +12,7 @@ type ErrorToast = {
...
-  <ToastMessage.Failure
-    minWidth="300px"
-    zIndex="2"
-    message={message}
-    secondaryMessage={secondaryMessage}
-    closeElem
-    closeFunction={closeToast}
-    className={styles.fadeInOut}
-  />
+  <Snackbar message={message + secondaryMessage} open={true} onClose={closeToast} className={styles.fadeInOut} />
```
[the remark this answers is not in the document]
Comment (macfarla) [reply src/components/Toasts/Error.tsx]: added a separator

File: src/components/EnodeTab/TextWithTooltip.tsx [hunk 5]
```diff
@@ -30,15 +29,16 @@ const TextWithTooltip: React.FC<TextWithTooltip> = ({ status, isAdmin, text }) =
...
         {text}
-      </Text>
+      </Typography>
     </Tooltip>
   ) : (
-    <Text className={styles.ellipsis} fontSize="14px">
-      {text}
-    </Text>
+    <Tooltip title={text} placement="bottom">
+      <Typography variant="body1" className={styles.ellipsis}>
```
[the remark this answers is not in the document]
Comment (macfarla) [reply src/components/EnodeTab/TextWithTooltip.tsx:38]: approximately

**Review by taccatisid — APPROVED** [review]

**Pull request merged**